afqinsight/nn/performance_utils.py

```diff
@@ -0,0 +1,328 @@
+import tempfile
+
+import numpy as np
+import tensorflow as tf
+import torch
+from neurocombat_sklearn import CombatModel
+from sklearn.impute import SimpleImputer
+from sklearn.model_selection import train_test_split
+
+from afqinsight import AFQDataset
+
+
+def prep_tensorflow_data():
```

Comment:
This could be rewritten to be a generally-useful helper function, but right now it's specialized for the HBN dataset. So, maybe make the dataset an input to the function instead.